Sources/ProtobufferBuilder/ProtobufferBuilder.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  File.swift
+//
+//
+//  Created by Nityananda on 26.11.20.
+//
+
+import Runtime
+
+public class ProtobufferBuilder {
+    private var messages: Set<Message> = .init()
+    private var services: Set<Service> = .init()
+    
+    public init() {}
+}
+
+public extension ProtobufferBuilder {
+    func add<T>(_ messageType: T.Type = T.self) throws {
```

Comment:
Currently, this ignores any functions the given type might have.
We will also need to check whether the given type conforms to `Component`, and if so populate the `private var services` (that already exists in `ProtobufferBuilder`) with the component and its `handle` method. 

This closely relates to my comment above, as here we would also have to distinguish between the different types of messages (`stream`s or unary) that the method takes as input and output. 

So, throwing in this 
```swift
struct Person: Component {
    let name: String
    let age: Int32

    public func handle() -> String {
        return "Hello \(name)."
    }
}
```
should produce a service like
```protobuf
service PersonService {
    rpc handle (Person) returns (/*...*/);
}

message Person {
    string name = 1;
    int32 age = 2;
}
```

Reply:
Conformance to `Component` is inforced in `ProtobufferSemanticModelBuilder`.
`ProtobufferBuilder.services` is now populated.

I created a follow-up issue: https://github.com/Apodini/Apodini/issues/51